daemon/api.go

```diff
@@ -283,8 +284,8 @@ func getPackagesInfo(c *Command, r *http.Request) Response {
 
 	for _, v := range bags {
 		m := v.Map(nil)
-		name := m["name"]
-		origin := m["origin"]
+		name, _ := m["name"].(string)
```

Comment:
Why is "name, _ := ..:" used instead of just "name := m["name"].(string)" when the "ok" value of the type conversion is not checked? Is it to avoid a panic if the type conversion fails? And if so, is this what we want?